From bb9f357d92025e8e5249e75330331f16153c2d41 Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Tue, 8 Mar 2016 14:32:34 -0600
Subject: [PATCH 32/37] ARM: dtb: Add gpio wlreg_on for wifi enablement

ARM: dtb: Add gpio wlreg_on signal for wifi enablement

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 arch/arm/boot/dts/imx6dqscm-qwks-rev1-wifi.dtsi | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dqscm-qwks-rev1-wifi.dtsi b/arch/arm/boot/dts/imx6dqscm-qwks-rev1-wifi.dtsi
index d6c2f18..6e6f021 100644
--- a/arch/arm/boot/dts/imx6dqscm-qwks-rev1-wifi.dtsi
+++ b/arch/arm/boot/dts/imx6dqscm-qwks-rev1-wifi.dtsi
@@ -18,9 +18,10 @@
                         compatible = "regulator-fixed";
                         regulator-min-microvolt = <5000000>;
                         regulator-max-microvolt = <5000000>;
-                        regulator-name = "wlreg_on";
-			regulator-boot-on;
-                        regulator-always-on;
+			regulator-name = "wlreg_on";
+                        gpio = <&gpio4 30 0>;
+                        startup-delay-us = <100>;
+                        enable-active-high;
                 };
         };
 
@@ -41,6 +42,7 @@
 				MX6QDL_PAD_SD2_DAT1__SD2_DATA1		0x17059
 				MX6QDL_PAD_SD2_DAT2__SD2_DATA2		0x17059
 				MX6QDL_PAD_SD2_DAT3__SD2_DATA3		0x17059
+				MX6QDL_PAD_DISP0_DAT9__GPIO4_IO30	0x13069
 			>;
 		};
 	};
-- 
1.9.1

